signer/Config.go

```diff
@@ -11,36 +16,231 @@ import (
 	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	"github.com/cosmos/cosmos-sdk/types/bech32"
+	"github.com/strangelove-ventures/horcrux/client"
 	"github.com/tendermint/tendermint/crypto"
+	"gopkg.in/yaml.v2"
 )
 
 type NodeConfig struct {
 	Address string
 }
 
+// Config maps to the on-disk JSON format
+type DiskConfig struct {
```

Comment:
The name `DiskConfig` makes me think we are configuring a disk. 

I think `signer.Config` is explanatory enough. 

In Go (and other languages), it's good to remember the name of the package contributes to the name as a whole.

Reply:
renamed